application_form/api/serializers.py

```diff
@@ -0,0 +1,80 @@
+from enumfields.drf import EnumField
+from rest_framework import serializers
+from rest_framework.fields import CharField, IntegerField, UUIDField
+
+from application_form.enums import ApplicationType
+from application_form.models import Applicant, Application
+from application_form.services import create_application
+from application_form.validators import SSNSuffixValidator
+
+
+class ApplicantSerializer(serializers.ModelSerializer):
+    date_of_birth = serializers.DateField(write_only=True)
+
+    class Meta:
+        model = Applicant
+        fields = [
+            "first_name",
+            "last_name",
+            "email",
+            "phone_number",
+            "street_address",
+            "city",
+            "postal_code",
+            "age",
+            "date_of_birth",
+            "ssn_suffix",
+        ]
+        extra_kwargs = {"age": {"read_only": True}}
+
+    def validate(self, attrs):
+        super().validate(attrs)
+        date_of_birth = attrs.get("date_of_birth")
+        validator = SSNSuffixValidator(date_of_birth)
```

Comment:
Good point, I added without checking. I have discussed it now and we agreed it would be better not to fully trust the data that comes from Drupal. However it should only add a warning log entry about an invalid SSN, not fail the whole request.